libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/CachedNavigationFeedbackEvent.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.core.telemetry.events
+
+/**
+ * A cache version of an internal MapboxTelemetry feedback event.
+ * It holds the required data for Feedback UI to display and to be updated with user input.
+ *
+ * @param feedbackId the key of the feedback
+ * @param feedbackType one of [FeedbackEvent.Type]
+ * @param description the user's additional comment about the feedback (optional)
+ * @param screenshot encoded screenshot (optional)
+ * @param feedbackSubType array of [FeedbackEvent.Description] (optional)
+ */
+class CachedNavigationFeedbackEvent internal constructor(
```

Comment:
`CachedNavigationFeedbackEvent` can be `internal` or moved into the `internal` package otherwise.

Reply:
this event should be exposed, because the UI side will need to use it. Please refer to the Feedback UX PR.

Reply:
It can be "public" but under the `internal` package so it'll be accessible from UI side.

> or moved into the `internal` package